test/java/seedu/addressbook/commands/ViewCommandTest.java

```diff
@@ -10,6 +10,10 @@
 
 import org.junit.Test;
 
+import seedu.addressbook.commands.Command;
```

Comment:
Do not change unrelated files in your PR, check your staged files before committing. Revert if necessary.